src/size.rs

```diff
@@ -172,13 +178,15 @@ impl Size {
     /// assert_eq!(size_neg.width, -4.0);
     /// assert_eq!(size_neg.height, -4.0);
     /// ```
+    ///
+    /// [rounded away]: FloatExt::expand
     #[inline]
     pub fn expand(self) -> Size {
         Size::new(self.width.expand(), self.height.expand())
     }
 
     /// Returns a new `Size`,
-    /// with `width` and `height` rounded down towards zero the nearest integer,
+    /// with `width` and `height` [rounded towards] zero to the nearest integer,
     /// unless they are already an integer.
```

Comment:
An optional suggestion - I'd slightly prefer this to be something like:
```suggestion
    /// with `width` and `height` [truncated][rounded towards] to their integer part.
    /// This effectively rounds them to the nearest integer towards zero.
```
I don't think "rounds towards zero to the nearest integer" flows very well, because in my mind "towards zero" is a modifier on the entire "rounds to the nearest integer".

Reply:
I'd rather this one be addressed separately as this sort of text is in more than one place.